src/main/xsl/docs.xsl

```diff
@@ -48,8 +48,9 @@
                 <xsl:text>My Documents</xsl:text>
             </h2>
             <form method="post" action="{links/link[@rel='upload']/@href}"
-                enctype="multipart/form-data">
-                <input name="file" type="file"/>
+                enctype="multipart/form-data"
+                onSubmit="if(document.getElementById('fileinput').value == '') return false;">
```

Comment:
`onSubmit="if(this.value== '') return false;"`
doesn't work... at least in Chrome